app/views/prison/visits/_issue_with_prisoner_section.html.erb

```diff
@@ -1,106 +1,142 @@
+        </label>
+      <% end %>
+      <%= error_container(f, :selection) do %>
+        <label class="block-label">
+          <%=
+            f.radio_button(
+              :selection, :prisoner_non_association,
+              class: 'js-Conditional'
+            )
+          %>
+          <%= t('.prisoner_non_association') %>
+        </label>
+      <% end %>
+    </div>
+    <div class="column-one-third">
+      <%= error_container(f, :selection) do %>
+        <label class="block-label">
+          <%=
+            f.radio_button(
+              :selection, :prisoner_moved,
+              class: 'js-Conditional'
+            )
+          %>
+          <%= t('.prisoner_moved') %>
+        </label>
+      <% end %>
+      <%= error_container(f, :selection) do %>
+        <label class="block-label">
+          <%=
+            f.radio_button(
+              :selection, :child_protection_issues,
+              class: 'js-Conditional'
+            )
+          %>
+          <%= t('.child_protection_issues') %>
+        </label>
+      <% end %>
+    </div>
+  </div>
+  <div id="no_allowance_details" class="panel panel-border-narrow">
+    <p><%= t('.no_allowance_tell_more') %></p>
+    <div class="grid-row push-top">
+      <div class="column-two-thirds">
+        <%=
+          single_field(
+            f, :allowance_will_renew, :check_box,
+            class: 'js-Conditional',
+            data: { 'conditional-el': 'allowance_will_renew_details' }
+          )
+        %>
+        <div id="allowance_will_renew_details" class="form-group">
+          <%= composite_field(f, :allowance_renews_on) do %>
+            <% %i[day month year].each do |field_name| %>
+              <div class="datefield form-group form-group-<%= field_name %>">
+                <label for="<%= f.object_name %>_allowance_renews_on_<%= field_name %>">
+                  <%= t(field_name, scope: 'date.fields') %>
+                </label>
+                <input
+                  class="form-control"
+                  value="<%=
+                    f.object.allowance_renews_on ?
+                      f.object.allowance_renews_on.send(field_name) : ''
+                    %>"
+                  type="number"
+                  name="<%= f.object_name %>[allowance_renews_on][<%= field_name %>]"
+                  id="<%= f.object_name %>_allowance_renews_on_<%= field_name %>"
+                  aria-describedby="allowance_renews_on"
+                />
+              </div>
+            <% end %>
+          <% end %>
+        </div>
+        <%=
+          single_field(
+            f, :privileged_allowance_available, :check_box,
+            class: 'js-Conditional',
+            data: { 'conditional-el': 'privileged_allowance_available_details' }
+          )
+        %>
+        <div id="privileged_allowance_available_details" class="form-group">
+          <%= composite_field(f, :privileged_allowance_expires_on) do %>
```

Comment:
I've modified this field name it use to be:
```ruby
<%= composite_field(f, :privileged_allowance_available_on) do %>
```
which is incorrect as the attribute is ```privileged_allowance_expires_on``` and was causing the validation errors not to appear.
I'm not sure if this could have any consequences elsewhere. @chrishanes ?